lua/drop/config.lua

```diff
@@ -202,9 +681,31 @@ end
 
 function M.get_theme()
   local theme = M.options.theme
+  local holidays = M.options.holidays
   if type(theme) == "string" then
     theme = M.themes[theme]
   end
+  -- Check if today is a holiday or falls within a holiday range.
+  if holidays ~= nil then
+    for holiday, date in pairs(holidays) do
+      if date ~= nil then
+        local startMonth, startDay = date.month or date.startDate.month, date.day or date.startDate.day
+        local endMonth, endDay = date.month or date.endDate.month, date.day or date.endDate.day
+        local isRange = startMonth ~= endMonth or startDay ~= endDay
+        if isRange and month >= startMonth and month <= endMonth then
+          if
+            (month == startMonth and day >= startDay)
+            or (month == endMonth and day <= endDay)
+            or (month > startMonth and month < endMonth)
+          then
+            theme = M.themes[holiday]
+          end
+        elseif not isRange and month == startMonth and day == startDay then
+          theme = M.themes[holiday]
+        end
+      end
+    end
+  end
```

Comment:
Augment the get_theme function to check if the current date falls within one of the provided/default holidays and override the theme if so. `holidays` option can be set to `nil` or `{}` to disable automatic holiday themes.